bin/options.js

```diff
@@ -1,41 +1,67 @@
 'use strict';
 
-/**
- * Dependencies.
- */
-
 var fs = require('fs');
 
-/**
- * Export `getOptions`.
- */
+var defaultOptionsPath = 'test/mocha.opts';
 
-module.exports = getOptions;
+var options = Object.create(null, {
```

Comment:
what's the necessity of `Object.create()` and the properties here?

Reply:
The aim is to collect and expose in a secure manner, the set of related _options_ units in a map. Using `Object.create` is a discipline. I believe there would be less JS bashing if the bashers applied this very discipline more.